base/markdown/Common/blockquote.jl

```diff
@@ -0,0 +1,52 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+type BlockQuote
+    content
+end
+
+BlockQuote() = BlockQuote([])
+
+# TODO: Laziness
+@breaking true ->
+function blockquote(stream::IO, block::MD)
+    withstream(stream) do
+        buffer = IOBuffer()
+        empty = true
+        while eatindent(stream) && startswith(stream, '>')
+            startswith(stream, " ")
+            write(buffer, readline(stream))
+            empty = false
+        end
+        empty && return false
+
+        md = takebuf_string(buffer)
+        push!(block, BlockQuote(parse(md, flavor = config(block)).content))
+        return true
+    end
+end
+
+function html(io::IO, md::BlockQuote)
+    withtag(io, :blockquote) do
+        println(io)
+        html(io, md.content)
+    end
+end
+
+function latex(io::IO, md::BlockQuote)
+    wrapblock(io, "quote") do
+        latex(io, md.content)
+    end
+end
+
+function plain(io::IO, md::BlockQuote)
+    print(io, "> ")
+    plain(io, md.content)
+end
```

Comment:
This should be implemented in a similar way to the ``term`` method below otherwise only the first line of the block will have ``> `` added. See below:


    julia> a = """
           > a
           > 
           > * 1
           > * 2
           >
           > ```
           > aaa
           > ```
           """
    "> a\n> \n> * 1\n> * 2\n>\n> ```\n> aaa\n> ```\n"

    julia> a |> Markdown.parse |> Markdown.plain |> print
    > a

      * 1
      * 2

    ```
    aaa
    ```